### DISCUSSION TITLE
Send MQTT commands to inverter

### DISCUSSION THREAD

**Discussion by rayenx** [jblance/mpp-solar#135] [Q&A]
Please,

Can anyone tell me how to setup to send commands to the inverter via MQTT? Do I need to setup the service first? 

Thanks.

**Comment by jblance** [comment]
Do you mean 'wait for a command on a mqtt topic and send that to the inverter' or do you have an Inverter that listens to mqtt commands? 

What inverter do you have? How are you connected?

**Reply by rayenx** [reply]
Hi,

I mean  'wait for a command on a mqtt topic and send that to the inverter'

I have an iSOLAR SML II from aexpert. I am able to read the values from the
inverter connected via USB to an OrangePi, but I want to set some parameters to it using MQTT.

Thanks.

**Reply by EmanuelLoos** [reply]
An MQTT listener would be great. I opened issue #288 regarding this a while ago. It would be very useful for automating charge/discharge based on hourly changing electricity prices like those from [aWATTar](https://www.awattar.at/).

**Comment by robwolff3** [comment]
This is being added in a future release according to https://github.com/jblance/mpp-solar/issues/228 and has instructions on how to use it now.

**Reply by rayenx** [reply]
Great to hear that!! Where I can find the instructions on how to use it?

Thanks.
